sdk/storage/azure-storage-blob-nio/src/main/java/com/azure/storage/blob/nio/AzureFileStore.java

```diff
@@ -118,10 +142,13 @@ public boolean supportsFileAttributeView(String s) {
     }
 
     /**
+     * This method always throws an {@code UnsupportedOperationException} as no {@link FileStoreAttributeView} is
+     * currently supported.
+     *
      * {@inheritDoc}
      */
     @Override
     public Object getAttribute(String s) throws IOException {
-        return null;
+        throw new UnsupportedOperationException("No FileStoreAttributeViews are currently supported.");
```

Comment:
```suggestion
        throw new UnsupportedOperationException("FileStoreAttributeViews aren't supported.");
```

Indicating currently isn't needed, let's be more blunt on the exception message.